core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/Parameter.java

```diff
@@ -40,8 +40,8 @@
 	private String required;
 	@JsonProperty
 	private String defaultValue;
-	@JsonProperty
-	private String choice;
+	@JsonProperty("enum")
+	private String[] choice;
```

Comment:
it was already mapped the same way: https://github.com/jbosstools/jbosstools-fuse/blob/9.2.x/tools/ide-catalog-generator/src/main/java/org/fusesource/ide/catalog/generator/model/eip/EIPProperty.java#L31